src/pru/main.c

```diff
@@ -75,25 +76,26 @@ timer_kick(void)
     timer_set(timer_read_time() + 50);
     CT_IEP.TMR_CMP_STS = 0xff;
     __delay_cycles(4);
-    CT_INTC.SECR0 = 1 << IEP_EVENT;
+    PRU_INTC.SECR0 = 1 << IEP_EVENT;
 }
 
 static uint32_t in_timer_dispatch;
 
 static void
 _irq_poll(void)
 {
-    uint32_t secr0 = CT_INTC.SECR0;
+    uint32_t secr0 = PRU_INTC.SECR0;
     if (secr0 & (1 << KICK_PRU1_EVENT)) {
-        CT_INTC.SECR0 = 1 << KICK_PRU1_EVENT;
+    //bit 19 was active
```

Comment:
I'm not sure what this comment means - is it dead code?

Reply:
Removed.